### PR TITLE
add 4th style sheet for newindex.html at wintel-branch

### PR DESCRIPTION
**Pull request by wintel2006**
thprd, can you validate the 4th section of the stylesheet for newindex?

### CODE REVIEW

File: newstyle.css [hunk 1]
```diff
@@ -272,3 +272,88 @@ a:hover {
...
+<style>/* Stylesheet 4: */
+body {
+    font: 100% Courier New;
+    margin: 20px;
+    line-height: 26px;
+    background-color: #000000;
+}
+
+.container {
+    xmin-width: 900px;
```
Comment (drpht) [inline newstyle.css]: xmin-width should be 800px instead.

Comment (wintel2006) [reply newstyle.css]: got it, check again

Comment (drpht) [reply newstyle.css]: i think it is good, and I am going to commit it now.

**Review by drpht — APPROVED** [review]
this is good change for the new index

**Pull request merged**